gluoncv/data/tracking_data/track.py

```diff
@@ -252,18 +252,18 @@ class TrkDataset(dataset.Dataset):
     """
     def __init__(self,
                  data_path=os.path.expanduser('~/.mxnet/datasets'),
-                 dataset_names=('vid', 'coco', 'det', 'yt_bb'),
-                 detaset_root=('vid/crop511', 'coco/crop511', 'det/crop511', 'yt_bb/crop511'),
-                 detaset_anno=('vid/train.json', 'coco/train2017.json', 'det/train.json',
-                               'yt_bb/train.json'),
-                 dataset_frame_range=(100, 1, 1, 3),
+                 dataset_names=('vid', 'yt_bb', 'coco', 'det'),
+                 detaset_root=('vid/crop511', 'yt_bb/crop511', 'coco/crop511', 'det/crop511'),
+                 detaset_anno=('vid/train.json', 'yt_bb/train.json', 'coco/train2017.json',
+                               'det/train.json'),
+                 dataset_frame_range=(100, 3, 1, 1),
                  dataset_num_use=(100000, -1, -1, -1),
                  train_search_size=255,
                  train_exemplar_size=127,
                  anchor_stride=8,
                  anchor_ratios=(0.33, 0.5, 1, 2, 3),
-                 train_base_size=8,
-                 train_output_size=25,
+                 train_base_size=0,
```

Comment:
What is train_base_size, and why it is 0?

Reply:
in pyost SiamRPN, train_base_size will help get output feature map size.Formula is 
```
score_size = (cfg.TRACK.INSTANCE_SIZE - cfg.TRACK.EXEMPLAR_SIZE) // \
            cfg.ANCHOR.STRIDE + 1 + cfg.TRACK.BASE_SIZE
```